kani-compiler/src/codegen_cprover_gotoc/compiler_interface.rs

```diff
@@ -339,6 +339,17 @@ fn print_report(ctx: &GotocCtx, tcx: TyCtxt) {
         details.";
         tcx.sess.warn(&msg);
     }
+
+    if !ctx.concurrent_constructs.is_empty() {
+        let mut msg = String::from(
+            "Kani does not support concurrency for now. The following constructs will be treated \
+            as sequential operations:\n",
+        );
+        for (construct, locations) in ctx.concurrent_constructs.iter() {
+            writeln!(&mut msg, "    - {} ({})", construct, locations.len()).unwrap();
```

Comment:
For upcoming Rust toolchain update 😊 
```suggestion
            writeln!(&mut msg, "    - {construct} ({})", locations.len()).unwrap();
```